unity-renderer/Assets/UIComponents/Tests/ImageComponentViewTests.cs

```diff
@@ -0,0 +1,78 @@
+using NUnit.Framework;
+using UnityEngine;
+
+public class ImageComponentViewTests
+{
+    private ImageComponentView imageComponent;
+
+    [SetUp]
+    public void SetUp() { imageComponent = BaseComponentView.Create<ImageComponentView>("Image"); }
+
+    [TearDown]
+    public void TearDown()
+    {
+        imageComponent.Dispose();
+        GameObject.Destroy(imageComponent.gameObject);
+    }
+
+    [Test]
+    public void ConfigureImageCorrectly()
+    {
+        // Arrange
+        ImageComponentModel testModel = new ImageComponentModel
+        {
+            sprite = Sprite.Create(new Texture2D(10, 10), new Rect(), Vector2.zero)
+        };
+
+        // Act
+        imageComponent.Configure(testModel);
+
+        // Assert
+        Assert.AreEqual(testModel, imageComponent.model, "The model does not match after configuring the button.");
+    }
+
+    [Test]
+    public void RefreshImageCorrectly()
+    {
+        // Arrange
+        Sprite testSprite = Sprite.Create(new Texture2D(10, 10), new Rect(), Vector2.zero);
+
+        imageComponent.model.sprite = testSprite;
+
+        // Act
+        imageComponent.RefreshControl();
+
+        // Assert
+        Assert.AreEqual(testSprite, imageComponent.model.sprite, "The sprite does not match in the model.");
+    }
+
+    [Test]
+    public void SetImageCorrectly()
+    {
+        // Arrange
+        Sprite testSprite = Sprite.Create(new Texture2D(10, 10), new Rect(), Vector2.zero);
```

Comment:
The `Texture2D` objects have to be destroyed manually to avoid leaks.

Reply:
Good catch! Done in: https://github.com/decentraland/unity-renderer/pull/1175/commits/531831692da512ed2d0095f4e2c0d08be70ee7be